### ISSUE TITLE
Not loading

### ISSUE THREAD

**Issue by lgshue** [VocalNutria9174/fnaf-2_html5#1]
It wont work

**Comment by ganonguy** [comment]
if you want to run it locally, i have a tutorial for this at humefogg.com/tutorial

**Comment by VocalNutria9174** [comment]
yeah, you have to host the page on a web server. you can use github, or even what ganonguy had sent.

sorry for the late response lol. i had randomly saw an email notification for this today, since im also not really active on github much anymore..
